docs/getting-started.md

```diff
@@ -51,25 +51,16 @@ pip install ./onefuzz*.whl
 ### Connecting to your instance
 
 Use the `onefuzz config` command to specify your instance of Onefuzz.
-This example uses the MSR hosted playground instance (only available
-to Microsoft employees).
 
 ```
-$ onefuzz config --endpoint https://onefuzz-playground.azurewebsites.net
+$ onefuzz config --endpoint https://$ONEFUZZ_INSTANCE_NAME.azurewebsites.net
 $ onefuzz versions check --exact
 "compatible"
 $
 ```
 
 From here, you can use Onefuzz.
 
-#### Using Onefuzz Playground
-
-Microsoft employees are welcome to use MSR's Playground instance of OneFuzz for
-experimentation purposes, but we reserve the right to stop or limit fuzzing
-jobs. [Come talk to us](https://aka.ms/fuzzingatmicrosoft) about running OneFuzz
-for your group!
```

Comment:
You probably want to move this message somewhere else in the getting started guide, since we have a number of Microsoft people looking to use 1f that don't know about "Fuzzing @ Microsoft"

Reply:
Yeah, I don't like doing that, but I agree that it would be most pragmatic.